Documents/UserManual/userManualFrench.md

```diff
@@ -0,0 +1,265 @@
+Vous pouvez le faire soit par Bluetooth, avec une carte NFC ou avec l'appli SportShield. <br><br>
+Par Bluetooth:
+
+- Activez votre connection Bluetooth sur votre smartphone.
+- Vous devrez trouver un dispositif nommé "SportShield".<br>
+<img src="./Img/findBluetooth.jpg" width="200"/>
+- Connectez votre smartphone avec le dispositif. <br>
+<img src="./Img/BluetoothSettings.png" width="200"/>
+<br><br>
+  
+Avec une carte NFC:
+
+- Faites toucher l'appareil NFC et le SportShield.
+
+A propos de l'appli SportShield, regardez la [partie dédiée](#app).
+
+<br>
+
+SportShield est maintenant set up, vous pouvez l'utiliser quand et où vous voulez.
+
+<div style="page-break-after: always;"></div>
+
+# Comment verrouiller SportShield
+
+SportShield protège vos affaires de sport contre les voleurs. Son installation n'a que quelques étapes:
+
+1. Tirez le câble de la sortie câble exit du SportShield.
+2. Entourez vos affaires avec le câble.
+3. Attachez le câble à l'autre embout.<br>
+<img src="./Img/cableHolderAndExitFrench.png" width="200"/>
+
+Si SportShield n'est pas connecté à votre smartphone ou avec une carte NFC, le dispositif ne fonctionnera pas comme préu.<br>
+Réferez vous à [partie set up](#set-up-sportshield).<br><br>
+SportShield est maintenant verrouillé et vous ne pourrez plus tirer sur le câble si vous ne le déverrouillez pas. <br><br>
+Vous recevrez une notification si le SportShield détecte un choc. <br>
+Pour plus d'informations sur les notifications, reportez-vous à la [section dédiée](#notifications).
+
+<div style="page-break-after: always;"></div>
+
+# Comment déverrouiller SportShield
+
+Le déverrouillage de SportShield est plutôt simple. Vous pouvez le déverrouiller par Bluetooth ou à l'aide de l'application. <br>
+Par Bluetooth:
+
+1. Allez dans vos paramètres Bluetooth.<br>
+<img src="./Img/BluetoothSettingsFunctionnalities.png" width="200"/>
+1. Clickez sur le boutton "Unlock" pour déverrouiller le SportShield.<br>
+<img src="./Img/BluetoothUnlock.png" width="200"/>
+
+Avec une carte NFC:
+
+- Faites toucher l'appareil NFC et le SportShield.
+
+A propos de l'app SportShield, reportez-vous à la [section dédiée](#app).
+
+<br>
+SportShield est maintenant déverrouillé. L'alarme ne sonnera pas et vous pouvez reprendre le dispositif.
+
+<br>
+
+<div style="page-break-after: always;"></div>
+
+# Alarme
+
+Quand SportShield est verrouillé, chaque choc détecté sonnera une alarme. <br>L'intensité de l'alarme dépend de l'intensité du choc: <br>
+
+- 3 sons légers lorsqu'un choc léger est détecté.
+  
+- 5 sons longs et aigus lorsqu'un choc important est détecté.
+<br><br>
+
+De plus, vous pouvez arrêter l'alarme quand vous le souhaitez. Pour ce faire, vous pouvez utiliser le Bluetooth ou l'application. <br>
+
+Par Bluetooth: ce sont les mêmes étapes que le [verrouillage du SportShield](#how-to-unlock-sportshield).
+
+Avec une carte NFC, ce sont les mêmes étapes que le [déverrouillage du SportShield](#how-to-unlock-sportshield).
```

Comment:
There's a mistake with the category link. And I imagine there will be several everywhere.